host.yaml

```diff
@@ -82,7 +82,7 @@ generator:
   race: 0
   # List of options that can be plando'd. Can be combined, for example "bosses, items"
   # Available options: bosses, items, texts, connections
-  plando_options: "bosses"
+  plando_options: "items"
```

Comment:
why is this change here?

Reply:
Had to change it during a beta test, accidentally committed. Will do another pass tonight for small things like this I've missed.